cxx/util_io.cc

```diff
@@ -36,22 +36,33 @@ T_schema load_schema(const std::string& path) {
   return schema;
 }
 
-T_observations load_observations(const std::string& path) {
+ObservationVariant observation_string_to_value(const std::string& value_str, 
```

Comment:
I think this code should live in the distributions/ directory, in the same file where we end up putting the get_distribution_by_name functionality.  I.e.: have a single place that gets changed when adding a new distribution (and possibly corresponding observation type).

Reply:
I'll leave this to a bigger refactor (we should break up hirm.hh and move definitions to .cc files) because as-is, it would create a circular dependency between the new file (which needs to include Relation for get_relation_by_name) and hirm.hh.